vm/systemSmartContracts/delegation.go

```diff
@@ -1430,6 +1456,307 @@ func (d *delegation) withdraw(args *vmcommon.ContractCallInput) vmcommon.ReturnC
 	return vmcommon.Ok
 }
 
+func (d *delegation) checkArgumentsForGeneralViewFunc(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
```

Comment:
:+1:  we can make this part of a common "something" that will be embedded in each system SC